src/relayclient/RelayClient.ts

```diff
@@ -15,9 +15,14 @@ import RelayedTransactionValidator from './RelayedTransactionValidator'
 import { configureGSN, getDependencies, GSNConfig, GSNDependencies } from './GSNConfigurator'
 import { RelayInfo } from './types/RelayInfo'
 
-// generate "approvalData" for a request. must return string-encoded bytes array
-export const EmptyApprovalData: AsyncApprovalData = async (): Promise<PrefixedHexString> => {
-  return await Promise.resolve('0x')
+export const ZERO_BYTES32 = '0x'.padEnd(66, '0')
+
+// generate "approvalData" and "paymasterData" for a request.
+// paymasterData must be valid bytes32 value (defaults to zero-bytes)
+// approvalData is string-encoded bytes array, (defaults to empty array)
+// either field can be missing, and filled with default
+export const EmptyPaymasterData: AsyncPaymasterData = async (): Promise<PaymasterData> => {
+  return await Promise.resolve({})
```

Comment:
There are exactly 0 reasons to put these 2 in one object in my view

Reply:
they are both dynamic parameters, per-request that are only visible by paymaster's `acceptRelayedCall`.
Why separate them?

Reply:
why unite them in interface?